core/blockchain.go

```diff
@@ -257,12 +257,23 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, genesis *Genesis
 		Journal:   cacheConfig.TrieCleanJournal,
 		Preimages: cacheConfig.Preimages,
 	})
-	// Setup the genesis block, commit the provided genesis specification
-	// to database if the genesis block is not present yet, or load the
-	// stored one from database.
-	chainConfig, genesisHash, genesisErr := SetupGenesisBlockWithOverride(db, triedb, genesis, overrides)
-	if _, ok := genesisErr.(*params.ConfigCompatError); genesisErr != nil && !ok {
-		return nil, genesisErr
+
+	var genesisHash common.Hash
+	var genesisErr error
+
+	if chainConfig.IsArbitrum() {
+		genesisHash = rawdb.ReadCanonicalHash(db, chainConfig.ArbitrumChainParams.GenesisBlockNum)
+		if (genesisHash == common.Hash{}) {
```

Comment:
There's an extra set of parens here

Reply:
Ah, I see it's for the `common.Hash` I think normally in the nitro codebase we express this as `if genesisHash == (common.Hash{}) {`